consensus/istanbul/validator/weighted.go

```diff
@@ -200,6 +199,7 @@ func NewWeightedCouncil(addrs []common.Address, rewards []common.Address, voting
 	if valSet.Size() > 0 {
 		valSet.proposer.Store(valSet.GetByIndex(0))
 	}
+	valSet.SetSubGroupSize(committeeSize)
```

Comment:
Why did you change the position of setting the sub group size? Is it related to the change of SetSubGroupSize function?

Reply:
I thought "SubGroupSize" needs to be assigned after the validator size is set.